blackbird_python/blackbird/listener.py

```diff
@@ -427,6 +446,28 @@ def exitStatement(self, ctx: blackbirdParser.StatementContext):
         else:
             self._program._operations.append(operation)
 
+    def enterForloop(self, ctx: blackbirdParser.ForloopContext):
+        self._infor = True
+
+    def exitForloop(self, ctx: blackbirdParser.ForloopContext):
+        self._infor = False
```

Comment:
Ah, it stands for 'in for loop' 😆 I would suggest changing it to `self._in_for`

Reply:
I kept reading it as 'info-r'

Reply:
Yep, I completely agree. It was just a quickly named variable for testing at first, and then I just forgot to update it to something more legible. :laughing: